od-store-query-vars.php

```diff
@@ -6,7 +6,7 @@
  * Requires at least: 6.5
  * Requires PHP: 7.2
  * Requires Plugins: optimization-detective
- * Version: 0.1.3
+ * Version: 0.2.0
```

Comment:
TODO: Add short-circuit if not on the required OD version.

Reply:
See https://github.com/WordPress/performance/commit/fb7e35c6c0d8e818cb91273fedeaeb84e3d0619b